src/anbox/cmds/container_manager.cpp

```diff
@@ -70,6 +71,15 @@ anbox::cmds::ContainerManager::ContainerManager()
         return EXIT_FAILURE;
       }
 
+      std::string log_dir = "/var/log/anbox";
+      if (!fs::exists(log_dir)) {
+        fs::create_directories(log_dir);
```

Comment:
Creating this only from the container manager is pretty racy. Which part of the code ever tries to write into this directory should attempt to create the directory. Also things running inside the user session (like session-manager and launch commands) should not log into /var/log but into $HOME.

Making things inside /var/log accessible for every user/group on the system is definitive wrong.